cmd/pitr/recoverer/recoverer.go

```diff
@@ -248,13 +260,16 @@ func (r *Recoverer) recover() (err error) {
 }
 
 func getLastBackupGTID(infoObj io.Reader) (string, error) {
```

Comment:
consider rename it to `getBackupGTID` because this function doesn’t stick to any `last` backup, which could be missleading

Reply:
it actually returns last GTID stored in backup